glusterapi/common.py

```diff
@@ -1,12 +1,23 @@
 import datetime
 import hashlib
+from uuid import UUID
 
 import jwt
 import requests
 
 from exceptions import GlusterApiError
 
 
+def validate_uuid(brick_id):
+    try:
+        UUID(brick_id, version=4)
```

Comment:
@Madhu-1 can you change the definition to validate_uuid(brick_id, version=4)
And use this as UUID(brick_id, version) . Not that we intend to change the version often but it is much cleaner.

Reply:
done